tests/run_tests/apps_run_test.py

```diff
@@ -109,7 +109,7 @@ def test_data_interpolation(tmp_path):
         ]:
             GEOH5.get_entity(uuid.UUID(uid))[0].copy(parent=workspace)
 
-    app = DataInterpolation(h5file=temp_workspace)
+    app = DataInterpolation(geoh5=temp_workspace)
     app.trigger.click()
```

Comment:
```suggestion
    app.trigger_click(None)
```

That way we get an actual error if it crashes. Otherwise ipywidgets absorbs the error in a try:except.